charts/flyte-agent/templates/agent/deployment.yaml

```diff
@@ -1,53 +1,55 @@
-{{- if .Values.flyteagent.enabled }}
+{{- if .Values.enabled }}
```

Comment:
Is this still necessary? If the chart is being installed, the intent is clear.

Reply:
Still necessary. For example, flyte agent is the `flyte-binary` dependency, but people can still disable it if they don't want to install the agent in their cluster. We disable the agent in the demo cluster in the CI.

Reply:
Would using a condition in Chart.yaml solve this problem?

Reply:
Just realized we can use condition in helm, will update it.